packages/core/src/components/Alert/index.tsx

```diff
@@ -42,173 +43,91 @@ export type Props = {
+        middleAlign={!children}
+        after={
+          onClose && (
+            <IconButton onClick={onClose}>
+              <IconClose
+                accessibilityLabel={T.phrase(
+                  'Close',
+                  {},
+                  { context: 'Close the alert', key: 'lunar.common.close' },
+                )}
+                size={unit * 3}
+              />
+            </IconButton>
+          )
+        }
+        before={
+          !hideStatusIcon &&
+          StatusIcon && <StatusIcon decorative color={iconColor} size={unit * 3} />
+        }
       >
-        <Row
-          middleAlign={!children}
-          after={
-            onClose && (
-              <IconButton onClick={onClose}>
-                <IconClose
-                  accessibilityLabel={T.phrase(
-                    'Close',
-                    {},
-                    { context: 'Close the alert', key: 'lunar.common.close' },
-                  )}
-                  size={unit * 3}
-                />
-              </IconButton>
-            )
-          }
-          before={
-            !hideStatusIcon &&
-            StatusIcon && <StatusIcon decorative color={iconColor} size={unit * 3} />
-          }
-        >
-          <Text bold>{title}</Text>
+        <Text bold>{title}</Text>
 
-          {children && (
-            <Spacing top={1}>
-              <Text>{children}</Text>
-            </Spacing>
-          )}
-        </Row>
-      </div>
-    );
-  }
+        {children && (
+          <Spacing top={1}>
+            <Text>{children}</Text>
+          </Spacing>
+        )}
+      </Row>
+    </div>
+  );
 }
 
-export default withStyles(
-  ({ color, unit, pattern }) => ({
-    alert: {
-      ...pattern.box,
-      display: 'block',
-      alignItems: 'start',
-      position: 'relative',
-      borderColor: color.core.neutral[1],
-      backgroundColor: color.accent.bg,
-      overflow: 'hidden',
-      padding: unit * 3,
-      wordBreak: 'break-word',
-
-      ':before': {
-        content: '" "',
-        backgroundColor: color.accent.text,
-        position: 'absolute',
-        top: 0,
-        left: 0,
-        width: unit / 2,
-        height: '100%',
-      },
-    },
-
-    alert_inline: {
-      display: 'inline-block',
-    },
-
-    alert_danger: {
-      ':before': {
-        backgroundColor: color.core.danger[3],
-      },
-    },
-
-    alert_info: {
-      ':before': {
-        backgroundColor: color.core.primary[3],
-      },
-    },
-
-    alert_notice: {
-      ':before': {
-        backgroundColor: color.core.secondary[3],
-      },
-    },
-
-    alert_success: {
-      ':before': {
-        backgroundColor: color.core.success[3],
-      },
-    },
-
-    alert_warning: {
-      ':before': {
-        backgroundColor: color.core.warning[3],
-      },
-    },
-
-    alert_statusIcon: {
-      paddingLeft: unit * 2.5,
-    },
-  }),
-  {
-    passThemeProp: true,
-  },
-)(Alert);
+Alert.propTypes = {
```

Comment:
If you define prop types like this, the `export default` will need to happen after, otherwise the TS declarations are broken.

Reply:
oh yeah, will update! just a few of these deviated from the typical conversion

Reply:
fixed!